modules/passkey/contracts/SafeWebAuthnSignerSingleton.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.0;
+
+import {SignatureValidator} from "./base/SignatureValidator.sol";
+import {IP256Verifier} from "./interfaces/IP256Verifier.sol";
+import {WebAuthn} from "./libraries/WebAuthn.sol";
+
+/**
+ * @title WebAuthn Safe Signature Validator Singleton
+ * @dev A contract that represents a WebAuthn signer.
+ * @custom:security-contact bounty@safe.global
+ */
+contract SafeWebAuthnSignerSingleton is SignatureValidator {
+    address public singleton;
+    uint256 public x;
+    uint256 public y;
+    IP256Verifier public verifier;
```

Comment:
Unfortunately, this will not work with 4337 :(

Reply:
This should resolve the problem https://github.com/safe-global/safe-modules/pull/370/commits/764804b9a8d0d4783b05533d8763faecf8cb3cea